networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters.
+
+[id="dependencies-network-observability"]
+== Dependencies of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki Operator: You must install Loki install before you install the Network Observability Operator. The Loki Operator is the backend that you can use to store all collected flows.
```

Comment:
```suggestion
* Loki Operator: You must provide a dedicated Loki instance before you install the Network Observability Operator. Loki is the backend that you can use to store all collected flows.
```

Reply:
@jpinsonneau, if we do that, my suggestion is far down in the text in the "small letters" section and not at the top.

Reply:
Well, we should not say that there's a dependency on "Loki Operator", but just on "Loki", and then we can mention the operator as a recommended way to install it

Reply:
Addressed- mentioned that it is recommended to install Loki Operator